### PR TITLE
deploy: don't requeue configs on stream updates yet

### PR DESCRIPTION
**Pull request by kargakis**
@mfojtik we still have the image change controller up and running so we don't need to do this just yet

I would like to have it for 1.3.

[test]

### CODE REVIEW

**Comment by mfojtik** [comment]
[test]

**Comment by mfojtik** [comment]
LGTM (approved for 1.3)

[merge]

File: pkg/deploy/controller/generictrigger/factory.go [hunk 1]
```diff
@@ -47,8 +47,9 @@ func NewDeploymentTriggerController(dcInformer, streamInformer framework.SharedI
 	c.dcStoreSynced = dcInformer.HasSynced
 
 	streamInformer.AddEventHandler(framework.ResourceEventHandlerFuncs{
```
Comment (liggitt) [inline pkg/deploy/controller/generictrigger/factory.go]: is there any point to adding an event handler with no functions registered?

Comment (kargakis) [inline pkg/deploy/controller/generictrigger/factory.go]: Not really. I can comment it out if you want

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**